pytorch/examples/SRNN/SRNN_Example.py

```diff
@@ -51,8 +51,24 @@
 epochs = config.epochs
 printStep = config.print_step
 valStep = config.val_step
+dropoutProbability_l0 = 0.2
+dropoutProbability_l1 = 0.2
 
-srnn2 = SRNN2(numInput, numClasses, hiddenDim0, hiddenDim1, cellType).to(device)
+print(cellType)
+'''
+cellArgs (optional) will be passed to the respective cell
+
+Example OPTIONAL args for FastGRNNCell
+cellArgs = {'gate_non_linearity':"sigmoid",'update_non_linearity':"tanh",
+				'wRank':None, 'uRank':None,'zetaInit':1.0, 'nuInit':-4.0, 
+				'batch_first':False}
+
+'''
+cellArgs = {}
+
+srnn2 = SRNN2(numInput, numClasses, hiddenDim0, hiddenDim1, cellType,
```

Comment:
Align arguments with braces. Install some PEP8 based linter on your editer (Sublime?) to automate this